draft-wood-cfrg-aead-limits.md

```diff
@@ -323,6 +341,87 @@ This results in reducing the limit on `v` by a factor of 2^64.
 v * 2^64 + (2l * (v + q))^2 <= p * 2^128
 ~~~
 
+# Multi-User AEAD Limits {#mu-limits}
+
+In the public-key, multi-user setting, {{MUSecurity}} proves that the success
+probability in attacking one of many independently users is bounded by the
+success probability of attacking a single user multiplied by the number of users
+present. Each user is assumed to have an independent and identically distributed
+key, though some may share nonces with some very small probability. Absent
+concrete multi-user bounds, this means the attacker advantage in the multi-user
+setting is the product of the single-user advantage and the number of users.
+
+This section summarizes the confidentiality and integrity bounds and limits for
+the same algorithms as in {{limits}}, except in the multi-user setting. The CL
+and IL values bound the total number of encryption and forgery queries (q and v).
+Alongside each value, we also specify these bounds.
+
+## AEAD_AES_128_GCM and AEAD_AES_256_GCM
+
+Concrete multi-user bounds for AEAD_AES_128_GCM and AEAD_AES_256_GCM exist
+due to {{?GCM-MU=DOI.10.1145/3243734.3243816}}.
+
+### Confidentiality Limit
+
+<!-- From (1) in {{GCM-MU}}, assuming n=2^7, \sigma = v*l, dropping the last term
+  (with denominator 2^(k+n), and dropping the first term since the adversary's
+  offline work dominates -->
+~~~
+CA <= ((v + q) * l * s) / 2^128)
+~~~
+
+This implies the following limit:
+
+~~~
+v + q <= (p * 2^128) / (l * s)
+~~~
+
+### Integrity Limit
+
+<!-- From Bad_8 advantage contribution to the inequality from 4.3 in {{GCM-MU}},
+  assuming \sigma = v*l -->
+~~~
+CA <= (1 / 2^1024) + ((2 * v) / 2^256) + ((2 * o * v) / 2^(k + 128))
+        + (128 * (v + (v * l)) / 2^k)
+~~~
+
+The last term in this inequality dominates. Thus, we can simplify this to:
```

Comment:
We need to explain why we think that this is the case, or what limits we place on different values.

Reply:
Good call. I'll add some color!